src/sql/src/session/vars.rs

```diff
@@ -1070,37 +1070,6 @@ impl Default for SystemVars {
 }
 
 impl SystemVars {
-    /// Set of [`SystemVar`]s that can also get set at a per-Session level.
-    ///
-    /// TODO(parkmycar): Instead of a separate list, make this a field on VarDefinition.
-    const SESSION_VARS: LazyLock<BTreeMap<&'static UncasedStr, &'static VarDefinition>> =
```

Comment:
Had to move this outside the `SystemVars` impl because associated `static` items are not allowed.